### PR TITLE
Add DataExplorer plugin 

### PR DESCRIPTION
**Pull request by alfonsoar**
*Issue #, if available:*

*Description of changes:*

Adding DataExplorer plugin

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by alfonsoar** [comment]
@claytonparnell @balajisankar15 @nikhilumeshsargur Could you review this? It has the same motivation as https://github.com/aws/sagemaker-distribution/pull/284

File: test/test_artifacts/v2/maxdome_data_explorer.test.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ARG COSMOS_IMAGE
```
Comment (claytonparnell) [inline test/test_artifacts/v2/maxdome_data_explorer.test.Dockerfile:1]: Can you update this to 
```
ARG SAGEMAKER_DISTRIBUTION_IMAGE
FROM $SAGEMAKER_DISTRIBUTION_IMAGE

Comment (alfonsoar) [reply test/test_artifacts/v2/maxdome_data_explorer.test.Dockerfile]: done, https://github.com/aws/sagemaker-distribution/pull/373/commits/dcda492365a6584bf0d00d8b4c5f152534125837

Do we also need to update the other ones?

* https://github.com/aws/sagemaker-distribution/blob/2.0.0-beta/test/test_artifacts/v2/maxdome_jupyter_server_extension.test.Dockerfile#L1-L2

* https://github.com/aws/sagemaker-distribution/blob/2.0.0-beta/test/test_artifacts/v2/maxdome_jupyter_session_manager.test.Dockerfile#L1-L2

Comment (claytonparnell) [reply test/test_artifacts/v2/maxdome_data_explorer.test.Dockerfile]: Ah yeah looks like it. Used to be COSMOS_IMAGE but we changed all the ones on `main` recently

Comment (alfonsoar) [reply test/test_artifacts/v2/maxdome_data_explorer.test.Dockerfile]: got it, i belive this is in the process of being reverted so not doing it in this PR.

**Review by nikhilumeshsargur — APPROVED** [review]
Approving the change

**Review by claytonparnell — APPROVED** [review]
Approving. please attach test results once they have been run

**Comment by alfonsoar** [comment]
waiting on resolution of: https://github.com/aws/sagemaker-distribution/pull/374

**Pull request closed without merging**